open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/security-sync-connectors/ranger-connector/pom.xml

```diff
@@ -55,6 +55,56 @@
             <artifactId>commons-collections4</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.ranger</groupId>
+            <artifactId>ranger-plugins-common</artifactId>
+            <exclusions>
+                <exclusion>
```

Comment:
interesting - nice idea to try and slim things down :-)

Reply:
I think it will be better to use the dependencyManagement from Egeria's pom.xml. I will update the PR. I saw you added this management of dependecies after I have created the PR.